vdb-bench-assembly/bower.json

```diff
@@ -31,6 +31,8 @@
     "angular": "~1.5",
     "angular-animate": "~1.5.5",
     "angular-route": "~1.5.5",
+    "angular-translate": "~2.13.0",
+    "angular-translate-loader-static-files": "~2.13.0",
```

Comment:
This is fine for versionings.
FYI: it implies >=2.13.0 < 2.14.0
This provides a good reference for how bower versionings work -> http://stackoverflow.com/questions/19019944/specifying-version-numbers-in-bower